text/src/autogluon/text/text_prediction/text_prediction.py

```diff
@@ -54,12 +54,33 @@ def default() -> dict:
             'search_options': None,        # Extra kwargs passed to searcher
             'scheduler_options': None,     # Extra kwargs passed to scheduler
             'time_limits': None,           # The total time limit
-            'num_trials': 4,               # The number of trials
-        }
+            'num_trials': 3,               # The number of trials
+        },
+        'seed': None,                      # The seed value
     }
     return ret
 
 
+@ag_text_prediction_params.register()
+def default_no_hpo() -> dict:
```

Comment:
Do we want to provide alternate HPs if the user specifies:
`hyperparameters='light'` or `'very_light'` or `'toy'`?

Reply:
I believe that this is unrelated to those presets, as this is in text_prediction

Reply:
Currently TextPrediction model must be specified explicitly, there is no default that uses it. I think that is the best approach for the moment until we hone in on when exactly it should be used.